report-jtreg-lib/src/test/java/io/jenkins/plugins/report/jtreg/ConfigFinderTest.java

```diff
@@ -0,0 +1,30 @@
+package io.jenkins.plugins.report.jtreg;
+
+import java.io.*;
+import java.nio.file.Files;
+
+public class ConfigFinderTest {
```

Comment:
Tyvm fort etst. The test hsould contain alo invalid xpath and invalid xml file negative tests. it may guide you on what to do wiuth those types of exceptions

Reply:
I extended the test.